File: 윤현철/JAVA/class210119/src/com/YHC/carApp/car/Car.java
package com.YHC.carApp.car;

public class Car {
	private String model;
	private String color;
	private int year;
	private String company;
	private String price;
	public Car(String model, String color, int year, String company,String price) {
		super();
		this.model = model;
		this.color = color;
		this.year = year;
		this.company = company;
		this.price = price;
	}
	public String getPrice() {
		return price;
	}
	public void setPrice(String price) {
		this.price = price;
	}
	public String getModel() {
		return model;
	}
	public void setModel(String model) {
		this.model = model;
	}
	public String getColor() {
		return color;
	}
	public void setColor(String color) {
		this.color = color;
	}
	public int getYear() {
		return year;
	}
	public void setYear(int year) {
		this.year = year;
	}
	public String getCompany() {
		return company;
	}
	public void setCompany(String company) {
		this.company = company;
	}
	public void printCarInfo() {
		System.out.println("모델	: " +model);
		System.out.println("색상	: " +color);
		System.out.println("년식	: " +year);
		System.out.println("제조사	: " +company);
		System.out.println("가격	: " +price);
		System.out.println("-----------------------------------------------------------");
	}
	
}

File: 이재창/CShap/20210215/P2/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace P2
{
    public class Car
    {
        public string Model { get; set; }
        public string Color { get; set; }
        public int Year { get; set; }
        public string Company { get; set; }

        public Car(string model, string color, int year, string company)
        {
            Model = model;
            Color = color;
            Year = year;
            Company = company;
        }

        public void PrintCarInfo()
        {
            Console.WriteLine($"모델명: {Model}");
            Console.WriteLine($"색상: {Color}");
            Console.WriteLine($"연식: {Year}");
            Console.WriteLine($"제조사: {Company}");
            Console.WriteLine();
        }
    }
}
